### ISSUE TITLE
Structure icon for add new taxonomic hierarchy

### ISSUE THREAD

**Issue by scratchpads** [NaturalHistoryMuseum/scratchpads2#6449]
- Added by [Nattarin](http://myriatrix.myspecies.info/user/968)
- Scratchpad: [Myriatrix](http://myriatrix.myspecies.info)

### Description:

I don't have structure icon at the upper side of page for add new taxonomic hierarchy in the future. Could you please fix this problem?

**Comment by therobyouknow** [comment]
Following up to see if this is still an issue, please can you advise?

**Comment by Archilegt** [comment]
I don't think that Nattarin is watching this repository. The issue was posted from within Myriatrix, during a training. For checking this, it is necessary to log in a test Scratchpad as a non-admin, and check if the Structure button is visible.

**Comment by therobyouknow** [comment]
is this still a problem? @Archilegt   Happy to keep the issue open and work on it for you, along with the other issues I'm looking at?   

We've had server issues some times over the last 6 months or more so I wanted to establish if this issue was a temporary problem caused by that, or if it was indeed a software bug in scratchpads that I actually need to fix.

**Comment by Archilegt** [comment]
This is still an issue, as experienced by non-admin users during a recent training. "Structure" should be displayed to non-admin "editor" users at least. Maybe there is a set of user types checkboxes for giving visualization and access rights to menus like "Structure", but if so, I don't know where they are located.
